### ISSUE TITLE
Changing any NPC's to playable character

### ISSUE THREAD

**Issue by ghost** [leszekd25/spellforce_data_editor#10]
There's any way to change any NPC to playable status in game, just like SF2? I've tried what i can, but all i got is only a "friendly" status and blue circle around characters.

**Comment by ghost** [comment]
For example I want to make some Skeletons and Wraiths playable from game, which is normal hostile to player, assign them to some fractions like Orcs, or just place them on map and play later

**Comment by leszekd25** [comment]
I think you need to change the unit's race to RacePlayer for the unit to be playable, but I'm not sure, as I have never tested it. If you're on SpellForce Discord server, there are many people who would probably know the answer to this question.

**Comment by ghost** [comment]
Changed, still no effects... What's the link for this discord, please? I'd like to join it :) Btw. I tried to mess up with clan settings - nothing changed, blue circle as it was, only behaviour for others NPC's are different.

**Comment by ghost** [comment]
Maybe somewhere is a flag, that controls what unit is controlable or not?

**Comment by leszekd25** [comment]
The invite to Discord is here: https://discord.gg/6P9wCj
If changing race doesn't help, then I think enabling unit control through gamedata is not possible, but I don't know for certain. Folks at Discord server know better than me.

**Comment by ghost** [comment]
Yes! I found a solution to this, but in game only by typing individually lines with the id's of the units...

There's a chance to add this option to editor?

Application:TakeFigureControl(figureNumber)
Allows you to control any unit already on the map. The unit friend/foe will still be the same, though. So it will still be attacked and will attack all units it was an enemy to before taking control of it.

Source: https://fearlessrevolution.com/viewtopic.php?t=8948

**Comment by KamilZochowski** [comment]
Effect:

![npc_playable_1](https://user-images.githubusercontent.com/32534905/115558571-2ba4d300-a2b3-11eb-9eac-72ab7f884ac8.png)
